test/integration/scheduler/scheduler_test.go

```diff
@@ -34,9 +34,13 @@ import (
 
 var _ = ginkgo.Describe("Scheduler", func() {
 	const (
-		instanceKey    = "cloud.provider.com/instance"
-		onDemandFlavor = "on-demand"
-		spotFlavor     = "spot"
+		instanceKey           = "cloud.provider.com/instance"
+		onDemandFlavor        = "on-demand"
+		spotFlavor            = "spot"
+		highPriorityClassName = "high-priority"
+		highPriority          = 100
+		lowPriorityClassName  = "low-priority"
+		lowPriority           = 10
```

Comment:
since they are only used in one test, lets define those priority constants inside the test.